ios/UARCTModule/UARCTMessageView.m

```diff
@@ -0,0 +1,175 @@
+/* Copyright Airship and Contributors */
+
+#import "UARCTMessageView.h"
+
+@interface UARCTMessageView()
+@property (nonatomic, strong) UAWKWebViewNativeBridge *nativeBridge;
+@property (nonatomic, strong) WKWebView *webView;
+@property (nonatomic, strong) UAInboxMessage *message;
+@property (nonatomic, strong) UADisposable *fetchMessagesDisposable;
+@end
+
+NSString *const UARCTMessageViewErrorMessageNotAvailable = @"MESSAGE_NOT_AVAILABLE";
+NSString *const UARCTMessageViewErrorFailedToFetchMessage = @"FAILED_TO_FETCH_MESSAGE";
+NSString *const UARCTMessageViewErrorMessageLoadFailed = @"MESSAGE_LOAD_FAILED";
+
+NSString *const UARCTMessageViewMessageKey = @"message";
+NSString *const UARCTMessageViewRetryableKey = @"retryable";
+NSString *const UARCTMessageViewErrorKey = @"error";
+
+@implementation UARCTMessageView
+
+- (instancetype) init {
+    self = [super initWithFrame:CGRectZero];
+    if (self) {
+        self.nativeBridge = [[UAWKWebViewNativeBridge alloc] init];
+        self.nativeBridge.forwardDelegate = self;
+        self.webView = [[WKWebView alloc] initWithFrame:self.bounds];
+        self.webView.navigationDelegate = self.nativeBridge;
+        self.webView.allowsLinkPreview = ![UAirship messageCenter].disableMessageLinkPreviewAndCallouts;
```

Comment:
does this mean that allows link preview defaults to YES since in the message center it defaults to NO? Is the switch something someone can or should flip from the js side?

Reply:
This is code from our built in viewcontroller. We could expose it if someone wants it.

Reply:
I guess then what I really mean is should it just have a hardcoded YES value here instead of delegating to something that will always say YES